static/js/src/advantage/subscribe/react/APICalls/usePostCustomerInfoForPurchasePreview.jsx

```diff
@@ -1,7 +1,7 @@
 import { useMutation } from "react-query";
 import { postCustomerInfoForPurchasePreview } from "../../../contracts-api";
 
-const usePostCustomerInfoAnon = () => {
+const usePostCustomerInfoForPurchasePreview = () => {
```

Comment:
I renamed the function to reflect the underlying API function name. What do you think @ClementChaumel ? Personally I find it much more descriptive.